### ISSUE TITLE
Markdown List Indenting

### ISSUE THREAD

**Issue by johnpneumann** [HappenApps/Quiver#454]
When you attempt add a secondary item to a list you end up having to delete the initial list hyphen and then tab and add a new hyphen. eg.

```
# correct
- Item 1
    - subitem
# what happens
- Item 1
-    subitem
```

To reproduce:
- Create a markdown cell
- Create a list item
- Hit tab
- Fail

Expected behavior:
Hitting tab indents the list rather than having to delete the list item and then tabbing in.

Version: 3.0.1 (78) - OSX 10.11.3

**Comment by aeroechelon** [comment]
I would love to see this change too. I think I've used Quiver for a long time that I've forgotten about this annoyance. I always end up hitting `SHIFT + A`, then `TAB` to indent to second level.

**Comment by lyricsboy** [comment]
Other apps indent/outdent with ⌘] and ⌘[

I'd love to see that adopted in Quiver.

**Comment by baileych** [comment]
Agreed.  I've also seen ⌘-Tab and ⌘-Shift-Tab,but being a frequent Markdown cell user, would happily adopt whatever convention is chosen.

**Comment by ghost** [comment]
Is there no way to do this?  I must be missing something, because that's really important for markdown.  I'm also looking for this feature in numbered lists--it should restart at one and indent.  I can't seem to figure it out. 

Version: 3.0.3 (88) os x 10.12.3

**Comment by GriffGeorge** [comment]
Would love to see this issue resolved. It is my only annoyance with this software, otherwise an awesome product. 

Version 3.0.5 (97), macOS 10.12.4

**Comment by kecosgrove** [comment]
I second. This would be a big deal for me.

**Comment by nyammy** [comment]
I agree to change behavior of single tab key in the list.
Because it works as expected in the lists in the standard text cells, and option + tab works as expected (decrease indent) even if in the markdown lists.

**Comment by wesmoncrief** [comment]
This would be very appreciated
